src/RecipeBot/Properties/Resources.resx

```diff
@@ -144,4 +144,7 @@
   <data name="RecipeTagEntriesController_DeleteTagAsync_Tag_0_with_Id_1_was_successfully_deleted" xml:space="preserve">
     <value>Tag '{0}' with id '{1}' was successfully deleted.</value>
   </data>
+  <data name="RecipeEntriesController_GetAllRecipesByTagAsync_No_saved_recipes_are_found_with_Tag_0_" xml:space="preserve">
+    <value>No saved recipes are found with the tag '{0}'.</value>
```

Comment:
Introduce additional entry with "No saved recipes are found with the tag id '{0}'. for clarity?